index.html

```diff
@@ -6,14 +6,1394 @@
+                                        data-full-date="May 7, 2020 - 16:02">16 hours ago</time>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                    <li class="u-col-xs-12 u-col-m-4 u-col-l-3" data-event-category="Homepage"
+                        data-event-action="Latest stories" data-event-label="Post" data-event-non-interaction="true">
+                        <div class="story story--smallMobile">
+                            <a class="lazy story-image lazyLoaded" sizes="(max-width: 516px) 100vw, 516px" width="516"
+                                height="270"
+                                href="https://thenextweb.com/hardfork/2020/05/07/reddits-greediest-traders-debate-shorting-tesla-ahead-of-musks-joe-rogan-interview/"
+                                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
+                                style="background-image: url(&quot;https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/05/Untitled-1-516x270.jpg&quot;);"></a>
+                            <h4 class="story-title">
+                                <a
+                                    href="https://thenextweb.com/hardfork/2020/05/07/reddits-greediest-traders-debate-shorting-tesla-ahead-of-musks-joe-rogan-interview/">
+                                    Reddit's greediest traders debate shorting Tesla ahead of Musk's Joe Rogan interview
+                                </a>
+                            </h4>
+                            <ul class="story-byline">
+                                <li><a href="https://thenextweb.com/author/davidcanellis/">David Canellis</a></li>
+                                <li>
+                                    <time datetime="2020-05-07T12:21:03+00:00" class="timeago"
+                                        data-full-date="May 7, 2020 - 15:21">17 hours ago</time>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                </ul>
+            </section>
+            <section class="deals investments">
+                <header class="deals-header">
+                    <h2>
+                        <a href="#" target="_blank" class="right-arrow">
+                            Latest funding rounds on Index.co
+                        </a>
+                    </h2>
+                    <ul>
+                        <li>
+                            <a href="#" class="deals-header-more">See more</a>
+                        </li>
+                        <li>
+                            <a href="#" class="deals-header-more">Get updates by email</a>
+                        </li>
+                        <li>
+                            <a href="#" class="deals-header-more">Add info</a>
+                        </li>
+                    </ul>
```

Comment:
When there are only three articles, the text to the side of latest funding should be below.

> Original page
![image](https://user-images.githubusercontent.com/57414789/81483109-4a749b80-9201-11ea-9108-5f8a28ad34be.png)

>Your page
![image](https://user-images.githubusercontent.com/57414789/81483122-54969a00-9201-11ea-99c4-8293bc9d395e.png)